tests/UI/pages/commonPage.js

```diff
@@ -235,7 +235,8 @@ module.exports = class CommonPage {
           value: option.value,
         })),
     );
-    options = await options.filter(option => textValue === option.textContent);
+
+    options = await options.filter(option => textValue === option.textContent.trim());
```

Comment:
Is this in the scope of this PR ? :smile:

Reply:
Not exactly but yes, to many spaces in a option value (for currency select).